src/store/settings/settings-reducers.ts

```diff
@@ -30,4 +31,8 @@ export const settingsReducers = createReducer<SettingsState>(settingsInitialStat
     ...state,
     selectedRpcUrl
   }));
+  builder.addCase(setIsReinstalled.success, state => ({
+    ...state,
+    isNotFirstAppLaunch: true
```

Comment:
maybe change the flag name to `isFirstAppLaunch`?   
it's hard to understand `not` flags